### PR TITLE
Filter organization wrt site_configurations on course creation

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
While course creation all organizations were listed down in the dropdown. This PR will restrict the organization list on the bases of site_configurations.

### CODE REVIEW

File: cms/djangoapps/contentstore/views/organization.py [hunk 1]
```diff
@@ -19,5 +20,14 @@ class OrganizationListView(View):
...
         """Returns organization list as json."""
         organizations = get_organizations()
-        org_names_list = [(org["short_name"]) for org in organizations]
+        # [COLARAZ_CUSTOM] 
+        # Restrict user from creating courses with non-authorized organizations
+        if not request.user.is_superuser:
+            site_orgs = configuration_helpers.get_current_site_orgs()
+            org_names_list = []
+            for org in organizations:
+                if ((org["name"] in site_orgs) or (org["short_name"] in site_orgs)):
```
Comment (danialmalik) [inline cms/djangoapps/contentstore/views/organization.py]:
```suggestion
                if org["name"] in site_orgs or org["short_name"] in site_orgs:
```

File: cms/djangoapps/contentstore/views/organization.py [hunk 2]
```diff
@@ -19,5 +20,14 @@ class OrganizationListView(View):
...
+                    org_names_list.append((org["short_name"]))
+        else:
+            org_names_list = [(org["short_name"]) for org in organizations]
```
Comment (danialmalik) [inline cms/djangoapps/contentstore/views/organization.py]:
```suggestion
            org_names_list = [org["short_name"] for org in organizations]
```

Comment (HamzaIbnFarooq) [reply cms/djangoapps/contentstore/views/organization.py]: done

File: cms/djangoapps/contentstore/views/organization.py (hunk 1 above)
Comment (HamzaIbnFarooq) [reply cms/djangoapps/contentstore/views/organization.py]: done

**Pull request merged**